### ISSUE TITLE
'custom' ceph_repository validation failed

### ISSUE THREAD

**Issue by zugwan** [ceph/ceph-ansible#3307]
**Bug Report**

What happened:
When 'custom' is used for ceph_repository, validation failed.
```
TASK [ceph-validate : validate provided configuration] ******************************************************************************************************************************
Thursday 08 November 2018  05:47:30 +0000 (0:00:00.631)       0:06:44.487 *****
 [ERROR]: [ceph-1] Validation failed for variable: ceph_repository

 [ERROR]: [ceph-1] Given value for ceph_repository: custom

 [ERROR]: [ceph-1] Reason: ceph_repository must be either 'community', 'rhcs' or 'dev'

fatal: [ceph-1]: FAILED! => {
    "changed": false,
    "msg": "[ceph-1] Validation failed for variable: ceph_repository\n[ceph-1] Reason: ceph_repository must be either 'community', 'rhcs' or 'dev'\n[ceph-1] Given value for ceph_repository: custom",
    "stderr_lines": [
        "[ceph-1] Validation failed for variable: ceph_repository"
    ]
}
```
'custom' is missing in validation process.
https://github.com/ceph/ceph-ansible/blob/640cad3fd810f0aacd41fc35b96f0be3f85fbd0d/plugins/actions/validate.py#L137

What you expected to happen:
Validation succeed

How to reproduce it (minimal and precise):

* all.yml
```
ceph_stable_release: luminous
ceph_origin: repository
ceph_repository: custom
ceph_custom_repo: "http://localrepo/ceph/ceph.repo"
```

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Docker version if applicable (e.g. `docker version`):
* Ansible version (e.g. `ansible-playbook --version`): 2.6.7
* ceph-ansible version (e.g. `git head or tag or stable branch`): d5409109fbec7a318fae09ad469f10ac0aae3866 / stable-3.2
* Ceph version (e.g. `ceph -v`):

**Comment by guits** [comment]
@andrewschoen why `custom` is not allowed ? Is it expected ? Or something we just missed in validate?

**Comment by dsavineau** [comment]
@guits I think this issue has been fixed by 6a79870

**Comment by guits** [comment]
@dsavineau yes, the backport was just missing, I triggered it few days ago 

see: https://github.com/ceph/ceph-ansible/pull/3593

I'll push a new tag soon

**Comment by dsavineau** [comment]
Fixed in v3.2.6
